services/vehicle/src/main/java/com/wedriveu/services/vehicle/entity/VehicleStore.java

```diff
@@ -1,15 +1,16 @@
-package com.wedriveu.services.vehicle.entity; /**
- * Created by Michele on 12/07/2017.
- */
+package com.wedriveu.services.vehicle.entity;
 
 import com.wedriveu.services.shared.utilities.Position;
+import io.vertx.core.eventbus.Message;
 
 import java.util.Date;
 import java.util.List;
 
 /**
- * @author Michele Donati
  * This inteface models the <em>com.wedriveu.services.vehicle.entity.Vehicle' database domain</em>.
+ *
+ * @author Michele Donati
+ * @author Marco Baldassarri
  */
 public interface VehicleStore {
```

Comment:
Why you modified the `VehicleStore`? Maybe you needed just a `Verticle` that would have use it?